marketplace-service/src/main/java/com/axonivy/market/controller/ImageController.java

```diff
@@ -54,4 +53,19 @@ public ResponseEntity<byte[]> findImageById(
     }
     return new ResponseEntity<>(imageData, headers, HttpStatus.OK);
   }
+
+  @GetMapping(BY_FILE_NAME)
+  public ResponseEntity<byte[]> findPreviewImageByName(
+      @PathVariable("imageName") String imageName) {
```

Comment:
@quanpham-axonivy please add desc for path variable as well since it's hard for users to know they must input full name including image extension. For example: @Parameter(description = "The image name", example = "slash-command.png"